app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlAdapter.kt

```diff
@@ -259,7 +265,10 @@ class SessionControlAdapter(
             PocketStoriesViewHolder.LAYOUT_ID -> return PocketStoriesViewHolder(
                 ComposeView(parent.context),
                 store,
-                components.core.client
+                components.core.client,
+                onPocketStoriesShown,
+                onCategoryClick,
+                onExternalLinkTapped
```

Comment:
We should just pass an Interactor here similar to `RecentTabViewHolder`

Reply:
Same as above. I'd try moving away from Controllers and especially from Interactors which currently don't do much.